desktop.css

```diff
@@ -159,12 +159,31 @@ body {
   flex-grow: 0;
 }
 
+.content-data > h2 {
```

Comment:
- [ ] Your application is not responsive, @sulaimanahmadi, I think you should better use the `min-width` property and make sure you style your app accordingly as for now, it is not respecting any of your CSS in big screens

![6](https://github.com/sulaimanahmadi/Portfolio/assets/77843050/b193363a-9aee-4328-bace-d0136ca33218)